api/image/image.go

```diff
@@ -24,24 +24,31 @@ func IsImageMatched(s, t string) bool {
 func Split(imageName string) (name string, tag string) {
```

Comment:
Can we change this method to return three things - a name, tag, and digest? Below in one of the tests I see `tag: :1.2.3@sha256:12345` which doesn't seem correct. We should differentiate the tag and digest where possible to help support cases where there are both.